test/plugins/test_embedart.py

```diff
@@ -47,21 +47,15 @@ class EmbedartCliTest(TestHelper, FetchImageHelper):
     abbey_differentpath = os.path.join(_common.RSRC, b"abbey-different.jpg")
 
     def setUp(self):
```

Comment:
This one's an interesting one - it doesn't seem to run `setup_beets` and thus doesn't seem to require the entire temporary directory / lib setup.

I think it only needs a few methods that `TestHelper` provides. Though now that you brought this to attention, I can see that this test class is actually being ignored:

```python
class EmbedartCliTest(TestHelper, FetchImageHelper):
```

It doesn't inherit from `unittest.TestCase`, so it doesn't get picked up. Will sort this out and check whether we have more cases like this one.

Reply:
It might not need the additional setup, but it might be worth it to inherit from `BeetsTestCase` just for consistency with other tests.  Otherwise, there should be a comment explaining why `BeetsTestCase` should _not_ be inherited from.

Reply:
See #5363 where I enabled this test and a couple of others that are skipped.

Reply:
@bal-e, maybe it would be a good idea to move the the functionality that this test needs away from `BeetsTestCase` to a `LibMixin` (or similar)? There are a couple of other tests (namely `test_query.py`) that are based from `BeetsTestCase` but they actually do not require the entire temporary dir setup which slows them down slightly.

Reply:
Ignore my message above - it actually requires the entire setup! I have now made it inherit from `BeetsTestCase`.